README.md

```diff
@@ -30,6 +30,9 @@ By default passing no option flags is equivalent to running keychain_dumper with
     
 ## Building
 
+### Demo
+![build and run gif demo](https://raw.githubusercontent.com/100apps/Keychain-Dumper/gh-pages/build-and-run.gif)
```

Comment:
Can you pull this gif down locally, add it to the repo, and then update the Readme to have a relative img url that points to it? Something like: https://help.github.com/articles/relative-links-in-readmes/